app/controllers/devise_sessions_controller.rb

```diff
@@ -139,7 +143,8 @@ def login_state
   end
 
   def do_sign_in
-    record_security_event(SecurityActivity::LOGIN_SUCCESS, user: login_state.user)
+    analytics_data = "from_confirmation_email" if params[:from_confirmation_email]
+    record_security_event(SecurityActivity::LOGIN_SUCCESS, user: login_state.user, analytics: analytics_data)
```

Comment:
Do we want this new field on all the events which may be logged while signing in?

Reply:
I'm not sure TBH.  We're only looking at login events right now, but we might as well add it since we may want to look at other metrics later on.